### PR TITLE
Mntor 3250

### PR DESCRIPTION
**Pull request by jespy2**
# References:

Jira: MNTOR-3250
Figma: https://www.figma.com/design/YqpVpUbSTzZMj9Jbp2po8Q/Mozilla-Monitor-Premium-LP?node-id=1942-12535&t=DnOFixpfMw7vf82s-0

# Description
New sub-navbar for the landing page (US users only)

# Screenshot (if applicable)
Screenshot 2024-06-20 at 7 01 08 AM

# How to test
Navigate to the landing page as a US user.  See the navbar just below the header.  Confirm that links work as expected.

# Checklist (Definition of Done)

- [x] Localization strings (if needed) have been added.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [ ] I've added or updated the relevant sections in readme and/or code comments
- [x] I've added a unit test to test for potential regressions of this bug.
- [ ] If this PR implements a feature flag or experimentation, the Ship Behind Feature Flag status in Jira has been set
- [x] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [x] All acceptance criteria are met.
- [x] Jira ticket has been updated (if needed) to match changes made during the development process.
- [ ] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

File: locales/en/landing-all.ftl [hunk 1]
```diff
@@ -4,6 +4,12 @@
 
 public-nav-name = { -brand-mozilla-monitor }
 
+# Top navbar
```
Comment (flodolo) [inline locales/en/landing-all.ftl:7]:
```suggestion
## Top navbar

```
This applies to all strings, so it should be a group-level comment.

And add an empty group-level comment after the 4 strings to reset it.

Comment (jespy2) [reply locales/en/landing-all.ftl]: Updated here:
https://github.com/mozilla/blurts-server/pull/4701/commits/23831f51c683c62a6c8af7b85d437e4e31f4169a

**Review by flozia — COMMENTED** [review]
Looking good already!

The main blocker would be that we don’t need to submit the introduced strings for localization — l’ve left a comment inline. I also noticed that we need to add some layout treatment for smaller viewports as well.
image

File: locales/en/landing-all.ftl [hunk 2]
```diff
@@ -4,6 +4,13 @@
 
 public-nav-name = { -brand-mozilla-monitor }
 
+## Top navbar
```
Comment (flodolo) [inline locales/en/landing-all.ftl:7]: Leave an empty line before and after group-level comments.

**Review by flodolo — APPROVED** [review]

File: locales/en/landing-all.ftl
[the remark this answers is not in the document]
Comment (jespy2) [reply locales/en/landing-all.ftl:1]: Updated here:
https://github.com/mozilla/blurts-server/pull/4701/commits/a1d11545915866abc485337c34b4c2a912de6fb1

File: src/app/(proper_react)/(redesign)/(public)/LandingView.tsx [hunk 3]
```diff
@@ -43,6 +43,7 @@ export const View = (props: Props) => {
     <>
       <AccountDeletionNotification />
       <main className={styles.wrapper}>
+        {props.countryCode === "us" && <TopNavBar l10n={props.l10n} />}
```
[the remark this answers is not in the document]
Comment (jespy2) [reply src/app/(proper_react)/(redesign)/(public)/LandingView.tsx]: Updated here:
https://github.com/mozilla/blurts-server/pull/4701/commits/20096dd49ad2ef50d8aace8c506af8cfb38bcdad

[the remark this answers is not in the document]
Comment (jespy2) [reply src/app/(proper_react)/(redesign)/(public)/LandingView.tsx]: Styles are updated here:
https://github.com/mozilla/blurts-server/pull/4701/commits/42113e410cfbf9b8d9047873704d6ec6d9b769c6

File: locales/en/landing-all.ftl (hunk 2 above)
Comment (jespy2) [reply locales/en/landing-all.ftl]: updated here:
https://github.com/mozilla/blurts-server/pull/4701/commits/7f80c24e71e8bc5cb29e7c7a148fd4021142c68d

**Review by flozia — APPROVED** [review]
Looks good to me — thanks!

**Pull request merged**